lib/active_record/connection_adapters/oracle_enhanced/schema_statements.rb

```diff
@@ -157,6 +157,15 @@ def indexes(table_name) #:nodoc:
           all_schema_indexes.select { |i| i.table == table_name }
         end
 
+        def columns(table_name)
+          table_name = table_name.to_s
+          if @columns_cache[table_name]
+            @columns_cache[table_name]
+          else
+            @columns_cache[table_name] = super(table_name)
+          end
+          @columns_cache[table_name]
```

Comment:
I think that this explicit return value line is unnecessary. The return value is determined by L163 or L165.

Reply:
Right. Updated.